crates/iota-core/src/authority.rs

```diff
@@ -4005,7 +4005,7 @@ impl AuthorityState {
                 };
                 return Ok(Some((
                     (*transaction).clone().into_message(),
-                    TransactionStatus::Executed(cert_sig, effects.into_inner(), events),
+                    TransactionStatus::Executed(cert_sig, Box::new(effects.into_inner()), events),
```

Comment:
Does this exist in the upstream change PR?

Reply:
No, but without it, it introduced a cargo check / CI warning on our side (with our CI checks settings)
Moving the enum variant into a Box removes that warning.

Reply:
The warning was about enum variants being very different in byte size